service-discovery/git/src/main/scala/restui/servicediscovery/git/GitProvider.scala

```diff
@@ -0,0 +1,23 @@
+package restui.servicediscovery.git
+
+import scala.util.Try
+
+import akka.actor.ActorSystem
+import com.typesafe.config.Config
+import org.slf4j.LoggerFactory
+import restui.servicediscovery.ServiceDiscoveryProvider
+import restui.servicediscovery.git.settings.Settings
+
+class GitProvider extends ServiceDiscoveryProvider {
+
+  private val logger = LoggerFactory.getLogger(classOf[GitProvider])
+
+  override def start(actorSystem: ActorSystem, config: Config, callback: ServiceDiscoveryProvider.Callback): Try[Unit] =
+    Try {
```

Comment:
why do you wrap the whole thing into Try?

Reply:
Copy/Paste

Reply:
Copy/paste